src/ocl/Makefile

```diff
@@ -30,8 +30,8 @@ else
   LIBS = -lOpenCL
 endif
 
-ocl-stream: main.cpp OCLStream.cpp
-	$(CXX) $(CXXFLAGS) -DOCL $^ $(EXTRA_FLAGS) $(LIBS) -o $@
+ocl-stream: ../main.cpp OCLStream.cpp
+	$(CXX) $(CXXFLAGS) -DOCL $^ $(EXTRA_FLAGS) $(LIBS) -o $@ -I. -I..
```

Comment:
The local copy of the OpenCL headers in `CL` have been put in `src/CL` but they should go in `src/ocl/CL` instead.